lingua_franca/parse.py

```diff
@@ -158,7 +159,33 @@ def extract_number(text, short_scale=True, ordinals=False, lang=None):
     return text
 
 
-def extract_duration(text, lang=None):
+def extract_calendar_duration(text, lang=None, replace_token=""):
+    """
+    Equivalent to extract_duration with
+
+        resolution=DurationResolution.RELATIVEDELTA_FALLBACK
+
+     Args:
+        text (str): string containing a duration
+        lang (str): the BCP-47 code for the language to use, None uses default
+        replace_token (str): string to replace consumed words with
+
+    Returns:
+        (relativedelta, str):
+                    A tuple containing the duration and the remaining text
+                    not consumed in the parsing. The first value will
+                    be None if no duration is found. The text returned
+                    will have whitespace stripped from the ends.
+
+    """
+    return extract_duration(text, lang,
+                            DurationResolution.RELATIVEDELTA_FALLBACK,
+                            replace_token)
+
+
+def extract_duration(text, lang=None,
+                     resolution=DurationResolution.TIMEDELTA,
+                     replace_token=""):
     """ Convert an english phrase into a number of seconds
```

Comment:
These docs are incorrect - this does not return a number of seconds (the new version has multiple types, and the old version returns a `timedelta`).